Src/spinlex.c

```diff
@@ -51,9 +52,9 @@ extern int	implied_semis, ltl_mode, in_seq, par_cnt;
 
 short	has_stack = 0;
 int	lineno  = 1;
-int	scope_seq[128], scope_level = 0;
+int	scope_seq[256], scope_level = 0;
```

Comment:
As stated earlier, if we are changing magic numbers, it would be nice to name them via defines. I see this is already done e.g., for MAX_TOKEN_TEXT_LENGTH

Reply:
Agree - but, it is not mine - this change is merged here from nimble repo before I've started any of my changes